articles/azure-stack/azure-stack-update-1903.md

```diff
@@ -153,26 +160,25 @@ The following are post-installation known issues for this build version.
 
 - An Ubuntu 18.04 VM created with SSH authorization enabled will not allow you to use the SSH keys to sign in. As a workaround, use VM access for the Linux extension to implement SSH keys after provisioning, or use password-based authentication.
 
-- Azure Stack now supports Windows Azure Linux agents greater than version 2.2.20. This support was a part of 1901 and 1902 hotfix and allows customers to maintain consistent linux images between Azure and Azure Stack.
-
+- Azure Stack now supports Windows Azure Linux agents greater than version 2.2.20. This support was a part of 1901 and 1902 hotfix and allows customers to maintain consistent Linux images between Azure and Azure Stack.
```

Comment:
Thank you, will fix.

Reply:
the original language is grammatically correct while the update language is not. this will not pass PR review as is. Suggested revision: 
> This support allows customers to maintain consistent Linux images between Azure and Azure Stack. It was added as part of the 1901 and 1902 hotfixes.

Reply:
Fixed :-)